packages/create-flex-plugin/templates/package.json

```diff
@@ -1,25 +1,37 @@
 {
-  "name": "{{pluginFileName}}",
+  "name": "{{name}}",
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "postinstall": "flex-check-start",
-    "prestart": "flex-check-start",
-    "start": "react-app-rewired start",
-    "build": "react-app-rewired build",
-    "test": "react-app-rewired test --env=jsdom",
-    "eject": "react-app-rewired eject"
+    "bootstrap": "flex-check-start",
```

Comment:
Can we get rid of this `flex-check-start` and just add the `index.html` to the public folder?

Reply:
I'd like to think about this a bit before doing that. We are talking about allowing third-party templates, so it would be nice that the index.html would live outside of the template.